aeat.py

```diff
@@ -1072,6 +1079,7 @@ def get_issued_sii_reports(cls):
                     ('sii_pending_sending', '=', True),
                     ('type', '=', 'out'),
                     ('move', '!=', None),
+                    ('invoice_date', '<=', today - timedelta(days=config.sii_default_limit_date)),
```

Comment:
79 cols.
Or we need to make "required" the config field, or ensure that it has value before set timedelta.
Use the new field name.